src/Index/Tests/Api/Rest/SelfReindexOperationTest.php

```diff
@@ -35,6 +35,7 @@ class SelfReindexOperationTest extends AbstractTest
     public static function setUpBeforeClass(): void
     {
         parent::setUpBeforeClass();
+        self::deleteElasticsearchFixtures();
```

Comment:
Same thing here, I really don't like muli-purpose commit :/

Reply:
it is an oversight, line removed